content/rdi/quickstart/write-behind-guide.md

```diff
@@ -229,6 +237,105 @@ To use the metrics you can either:
 
 - Scrape the metrics using RDI Prometheus exporter
 
+### Read-through jobs
+
+Read-through jobs are a mandatory part of the read-through pipeline configuration.
+
+In the jobs directory, which is located parallel to the `config.yaml` file, you should include a YAML file for each key pattern you want to read into the cache from a downstream database table in the event of a cache miss.
```

Comment:
, which is located adjacent to the `config.yaml` file,

Reply:
done